.github/workflows/cd.packages-stable.create-release-drafts.yml

```diff
@@ -0,0 +1,196 @@
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/
+
+      - name: Check package versions
+        run: ./tasks/check-package-version.sh ethereum-contracts SHOULD_PUBLISH_ETHEREUM_CONTRACTS ETHEREUM_CONTRACTS_NEW_VERSION >> "$GITHUB_ENV"
+
+      - name: Create ethereum-contracts stable release draft
+        if: env.SHOULD_PUBLISH_ETHEREUM_CONTRACTS == 1
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ethereum-contracts@v${{ env.ETHEREUM_CONTRACTS_NEW_VERSION }}
+          release_name: ethereum-contracts@v${{ env.ETHEREUM_CONTRACTS_NEW_VERSION }}
+          body_path: .github/RELEASE_TEMPLATE/release-notes.md
+          draft: true
+
+  create-subgraph-release-draft:
+    name: Create Subgraph Release Draft
+
+    runs-on: ubuntu-latest
+
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/
+
+      - name: Check package versions
+        run: ./tasks/check-package-version.sh subgraph SHOULD_PUBLISH_SUBGRAPH SUBGRAPH_NEW_VERSION >> "$GITHUB_ENV"
+
+      - name: Create subgraph stable release draft
+        if: env.SHOULD_PUBLISH_SUBGRAPH == 1
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: subgraph@v${{ env.SUBGRAPH_NEW_VERSION }}
+          release_name: subgraph@v${{ env.SUBGRAPH_NEW_VERSION }}
+          body_path: .github/RELEASE_TEMPLATE/release-notes.md
+          draft: true
+
+  create-and-publish-metadata-package:
+    name: Create Metadata Release Draft and Publish to NPM
+
+    runs-on: ubuntu-latest
+
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/
+
+      - name: Check package versions
+        run: ./tasks/check-package-version.sh metadata SHOULD_PUBLISH_METADATA METADATA_NEW_VERSION >> "$GITHUB_ENV"
+
+      - name: Create metadata stable release draft
+        if: env.SHOULD_PUBLISH_METADATA == 1
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: metadata@v${{ env.METADATA_NEW_VERSION }}
+          release_name: metadata@v${{ env.METADATA_NEW_VERSION }}
+          body: |
+            Please refer to the CHANGELOG.md file in metadata package for more details on the changeset.
+          draft: true
```

Comment:
do we still want to keep it draft stage? Since it's going to be published to npm nonethless.

Reply:
true, will set this to false

Reply:
that's default I believe